modules/mono/editor/GodotTools/GodotTools/GodotSharpEditor.cs

```diff
@@ -171,7 +179,7 @@ public void ShowErrorDialog(string message, string title = "Error")
         [UsedImplicitly]
         public Error OpenInExternalEditor(Script script, int line, int col)
         {
-            var editorId = (ExternalEditorId)(int)_editorSettings.GetSetting("mono/editor/external_editor");
+            var editorId = (ExternalEditorId)(int)_editorSettings.GetSetting(Settings.ExternalEditor);
```

Comment:
Users coming from previous betas will lose their settings.

Reply:
How do you think this should be handled?

Reply:
I don't know if we have something in Godot to handle editor setting renames. If we don't, we may want to still look for the old name and, if found, set the new setting's value; at least for the betas/RCs.

cc @akien-mga

Reply:
Unless there's a built-in way to do this, I think it may be too much work to bother with it. Otherwise, you could check if the old setting is present, move its value to the new one, and remove the old one to avoid repeating this.

Reply:
Can you be more specific about what you mean by "not bother with it"? Are you implying we'd leave everything labelled "mono" or that we could afford to break settings compatibility between betas?

Reply:
Sorry, I meant not bother about migrating from the old setting. It may be annoying, but it's a one-time thing and there are other breaking changes coming as well.